main.cpp

```diff
@@ -84,43 +104,83 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+int addTheNumbers(int num1, int num2, int num3 = 2)
+{
+    ignoreUnused(num1, num2, num3);
+    return{};
+}
 /*
  2)
  */
-
+bool openDoor(bool needToOpen, bool isLocked = true)
+{
+    ignoreUnused(isLocked, needToOpen);
+    return{};
+}
 /*
  3)
  */
-
+int remainingDistance(int MPG, int fuel = 12)
```

Comment:
This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```
The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`
- [ ] revise